### PR TITLE
Remove unused CSS

### PR DESCRIPTION
**Pull request by robertknight**
~~_**This is blocked on https://github.com/hypothesis/h/pull/3737**_~~

Remove unused CSS rules. These were found by using used-css-classes [1]
and find-unused-css-classes [2]:
```
  used-css-classes h/templates/client/*.html > used-classes.txt
  find-unused-css-classes used-classes.txt build/styles/app.css > unused-css-classes
```
And then manually checking classes in `unused-css-classes` to verify
that the classes were not referenced in first or third-party code, since
the tools only extract used class names from templates, the `used-css-classes`
tool also only looks at the `class` attribute of HTML tags, so it can miss
class names referenced in `ng-class` directives etc.

[1] https://github.com/robertknight/used-css-classes
[2] https://github.com/robertknight/find-unused-css-classes

### CODE REVIEW

File: h/templates/client/annotation.html [hunk 1]
```diff
@@ -66,7 +66,7 @@
     </excerpt>
   </section>
 
-  <!-- / Excerpts -- >
+  <!-- / Excerpts -->
```
Comment (robertknight) [inline h/templates/client/annotation.html]: This fixes a parse error with BeautifulSoup, which the used-class extraction tool uses. See first commit.

**Comment by nickstenning** [comment]
I don't think we can merge this until after we've shipped the site-v2 bundle to all users. Many pages in `h` still use styles from the app.

**Comment by robertknight** [comment]
Ah, damn it. You're right. Perhaps we could avoid this problem by just building the bundle and then adding it to the service repo as a checked-in file, since we're unlikely to be changing these styles.

**Comment by robertknight** [comment]
I created a PR for the Hypothesis site to remove the dependence on the app.css bundle from the client for site pages: https://github.com/hypothesis/h/pull/3737 . We can merge this once that PR has shipped.

**Comment by nickstenning** [comment]
LGTM.

**Pull request merged**